src/fancyprint/printer.py

```diff
@@ -3,8 +3,10 @@
 #######################
 import os
 import sys
+from typing import Tuple
```

Comment:
I don't see this library being used, anywhere else in code. Please remove it.

Reply:
Sorry that is my bad i forgot to remove it